utilities/common.py

```diff
@@ -14,7 +14,11 @@
 axis_eta = hist.axis.Regular(48, -2.4, 2.4, name = "eta")
 axis_pt = hist.axis.Regular(29, 26., 55., name = "pt")
 #ptV_binning = [0, 2, 3, 4, 4.75, 5.5, 6.5, 8, 9, 10, 12, 14, 16, 18, 20, 23, 27, 32, 40, 55, 100]
-ptV_binning = [0, 2, 3, 4, 5, 6, 7, 8, 9, 10, 12, 14, 16, 18, 20, 23, 27, 32, 40, 55, 100]
+#ptV_binning = [0, 2, 3, 4, 5, 6, 7, 8, 9, 10, 12, 14, 16, 18, 20, 23, 27, 32, 40, 55, 100]
+## 5% quantiles from aMC@NLO used in SMP-18-012
+#ptV_binning = [0.0, 1.971, 2.949, 3.838, 4.733, 5.674, 6.684, 7.781, 8.979, 10.303, 11.777, 13.435, 15.332, 17.525, 20.115, 23.245, 27.173, 32.414, 40.151, 53.858, 13000.0]
+## 10% quantiles from aMC@NLO used in SMP-18-012 with some rounding <== This one worked fine with toys
+ptV_binning = [0.0, 2.95, 4.73, 6.68, 8.98, 11.78, 15.33, 20.11, 27.17, 40.15, 120]
```

Comment:
Just introduce a new variable, ptV_10quantiles_binning or something. Or delete the old one, but no reason to leave commented out stuff

Reply:
fixed by the new commit [1f75b3e](https://github.com/WMass/WRemnants/pull/71/commits/1f75b3ef9eaaf3c6b174362ab1df5fb618ebb267)